doc/source/user_guide/ug_case_settings.rst

```diff
@@ -314,8 +314,10 @@ forcing_nml
    :widths: 15, 15, 30, 15 
 
    "", "", "", ""
-   "``atmbndy``", "``constant``", "bulk transfer coefficients", "``default``"
-   "", "``default``", "stability-based boundary layer", ""
+   "``atmbndy``", "string", "bulk transfer coefficients", "``similarity``"
+   "", "``similarity``", "stability-based boundary layer", ""
```

Comment:
Maybe here we could add something like
```suggestion
   "", "``similarity``", "stability-based boundary layer (previously: `default`)", ""
```

to help users that could be confused by their existing namelist having a now undocumented `default` value ?

Reply:
From CICE this is automatically changed from 'default' to 'similarity' together with a WARNING message printed to the log/stdout.